nextcloud/photos
# [stable27] fix: show error message for failing upload

How to reproduce

- Nextcloud 27 or 26
- Delete the photos directory in your home directory
- Try uploading a photo in the album view
- It will fail, because the photos directory does not exist but no error message is shown

Photos for Nextcloud 28 will create the missing folder automatically.
I decided against a backport because it was bigger refactoring. 

Before:

[Screencast from 2024-07-22 11-17-36.webm](https://github.com/user-attachments/assets/257d5c6e-f967-494a-89b2-c322fc8a2f73)

After:

[Screencast from 2024-07-22 11-15-48.webm](https://github.com/user-attachments/assets/47dade22-b5c2-48ae-952a-5280f4fcb91e)
